addons/website_sale/static/src/js/website_sale.js

```diff
@@ -22,6 +22,12 @@ publicWidget.registry.websiteSaleCartLink = publicWidget.Widget.extend({
         this._super.apply(this, arguments);
         this._popoverRPC = null;
     },
+    /**
+     * @override
+     */
+    willStart() {
+        return Promise.all([this._super.apply(this, arguments), this._updateCartQuantityValue()]);
```

Comment:
I think we talked about it in person but I don't remember why it wasn't required before.
Could you refresh my memory? 🙂

Reply:
This allows to get the CartQuantity when the page is reloaded from history (eg using the back button).

Sadly, it cannot be handled through an event from the Page Visibility Lifecycle and factorized with the visibilitychange event.